package.json

```diff
@@ -2,35 +2,30 @@
   "name": "@jupiterone/integration-sdk",
   "version": "0.15.0",
   "description": "The SDK for developing JupiterOne integrations",
-  "main": "dist/framework/index.js",
-  "types": "dist/framework/index.d.ts",
+  "main": "src/framework/index.js",
+  "types": "src/framework/index.d.ts",
   "repository": "git@github.com:JupiterOne/integration-sdk.git",
   "author": "JupiterOne <dev@jupiterone.io>",
   "license": "MPL-2.0",
   "engines": {
-    "node": "10.x || 12.x"
+    "node": "10.x || 12.x || 14.x"
```

Comment:
Seems the build matrix needs to cover this then, eh?

Reply:
Personally, I don't think we need to even specify the engine... I had just happen to notice that v14 of node was released recently so I didn't think we should block that from working.

Reply:
I wonder if this could be improved with something like ">10", since probably we just mean to say we aren't supporting known incompatibilities (if there are any).